### PR TITLE
Consolidate queue configs

### LINKED ISSUE #288: Queue config refactor and review
Making sure all configurations are consistent and based on queue usecase make sure the retry and all other options make sense. Duplication checks and etc.

* The configuration between producer and consumer should be consistent
* Suggestion: Using `DynamicModule` to register queues at one common module and imported in respective module

more details:
https://api.docs.bullmq.io/classes/v4.Queue.html

### PR DESCRIPTION
**Pull request by JoeCap08055**
For `account-service` and `content-publishing-service`:
- Reorganize imports so that we can import using '#lib/<module>' prefix instead of '../../../../lib/common/src/<module>/...'
- Refactor BullMQ connection & queue definitions to a common module for each app
    - Avoids multiple instantiation of `BullModule`
    - Eliminates possible errors in defining the same queue in multiple modules with different defaults

Closes #288 #316

### CODE REVIEW

File: services/account/libs/common/src/queues/queues.module.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { ConfigModule } from '#lib/config/config.module';
+import { ConfigService } from '#lib/config/config.service';
+import { BullModule } from '@nestjs/bullmq';
+import { DynamicModule, Global, Module } from '@nestjs/common';
+import Redis, { RedisOptions } from 'ioredis';
+import * as QueueConstants from './queue.constants';
+
+@Global()
+@Module({
```
Comment (claireolmstead) [inline services/account/libs/common/src/queues/queues.module.ts:9]: Nice!

File: services/content-publishing/apps/api/src/controllers/v1/asset.controller.v1.ts [hunk 2]
```diff
@@ -22,6 +32,7 @@ export class AssetController {
     description: 'Asset files',
     type: FilesUploadDto,
   })
+  @ApiResponse({ status: '2XX', type: UploadResponseDto })
```
Comment (claireolmstead) [inline services/content-publishing/apps/api/src/controllers/v1/asset.controller.v1.ts:35]: what does this status mean?

**Review by claireolmstead — APPROVED** [review]
So much refactoring, it's hard to read these changes. Looks like they're all mostly names changes for imports and some rearranging, ship it! If there's any logic changes that you want me to look at, would you mind tagging me?

**Comment by JoeCap08055** [comment]
> So much refactoring, it's hard to read these changes. Looks like they're all mostly names changes for imports and some rearranging, ship it! If there's any logic changes that you want me to look at, would you mind tagging me?

Sorry, I should have been more clear--no logic changes here, just refectoring of imports for shared config use and readability.

**Pull request merged**